x/wasm/keeper/keeper_test.go

```diff
@@ -1489,9 +1380,9 @@ func TestClearContractAdmin(t *testing.T) {
 	}
 }
 
-func TestClearContractAdminInactiveContract(t *testing.T) {
+func TestExecuteInactiveContract(t *testing.T) {
```

Comment:
Change the name to what tells the test expects the execution fails. (This test tries to change the contract status via non-gov and fails, right?)

Reply:
right, I've edited it to have a more precise meaning. If there is a more correct sentence, please feel free to suggest it.
thks!